patches/chrome-installer-mac-sign_chrome.py.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/chrome/installer/mac/sign_chrome.py b/chrome/installer/mac/sign_chrome.py
+index fa3a88200f5156975d0da5bcc52ed6f9896e792e..9068adc58127d24f50f47b3f1f939e2b6feaef69 100755
+--- a/chrome/installer/mac/sign_chrome.py
++++ b/chrome/installer/mac/sign_chrome.py
+@@ -28,6 +28,7 @@ def create_config(config_args, development):
+         An instance of |model.CodeSignConfig|.
+     """
+     config_class = config.CodeSignConfig
++    """
```

Comment:
This section needs to be commented out because `identity` and `keychain` below used to be passed into `create_config` as parameters, but now are packed into `config_args` parameter. Chromium probably never runs into error here because they successfully import their secret internal config and don't have exception in the `try`.

Reply:
I think we can still use this as you do in the [sign_brave.py](https://github.com/brave/brave-core/blob/master/build/mac/sign_brave.py). I'm porting their recent changes from `sign_chrome.py` to your `sign_brave.py` and I think that will work as you did before which is basically removing the section where they do the import of their config.

Reply:
It's the section where they do the import of their config that I commented out here in the patch, not the config_class, github's snippet isn't helping here.